integration/internal/dctest/cmd.go

```diff
@@ -0,0 +1,96 @@
+package dctest
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"sync"
+	"testing"
+
+	"github.com/concourse/concourse/integration/internal/cmdtest"
+	"github.com/concourse/concourse/integration/internal/ypath"
+	"github.com/stretchr/testify/require"
+)
+
+type Cmd struct {
+	cmdtest.Cmd
+}
+
+func Init(t *testing.T, composeFile string, overrides ...string) Cmd {
+	name := filepath.Base(t.Name())
+
+	files := append([]string{composeFile}, overrides...)
+
+	dc := cmdtest.Cmd{
+		Path: "docker-compose",
+		Env: []string{
+			"COMPOSE_FILE=" + strings.Join(files, ":"),
+			"COMPOSE_PROJECT_NAME=" + name,
+		},
+	}
+
+	for _, env := range os.Environ() {
+		if strings.HasPrefix(env, "TEST_") {
+			dc = dc.WithEnv(env)
+		}
+	}
+
+	// clean up docker-compose when the test finishes
+	cleanupOnce(t, func() {
+		if t.Failed() {
+			err := os.Mkdir("logs", os.ModePerm)
```

Comment:
```suggestion
			err := os.MkdirAll("logs", os.ModePerm)
```

Getting errors when it already exists